moirai-typesafeconfig/src/test/scala/com/nike/moirai/typesafeconfig/TypesafeConfigUserDecidersSpec.scala

```diff
@@ -86,6 +86,24 @@ class TypesafeConfigUserDecidersSpec extends FunSpec with Matchers {
     }
   }
 
+  describe("A featureEnabled config decider") {
+    val resourceLoader = FileResourceLoaders.forClasspathResource("moirai.conf")
+
+    val featureFlagChecker = ConfigFeatureFlagChecker.forConfigSupplier[Config](
+      Suppliers.supplierAndThen(resourceLoader, TypesafeConfigReader.FROM_STRING),
+      TypesafeConfigDecider.FEATURE_ENABLED
+    )
+
+    it("should be enabled if the configuration say so") {
+      featureFlagChecker.isFeatureEnabled("coffee") shouldBe true
+      featureFlagChecker.isFeatureEnabled("tea") shouldBe false
```

Comment:
Let's add a case where we do provide `FeatureCheckInput` for a user. The library is meant to allow it to be possible that the code doing the check doesn't know whether or not the input is relevant to decision, so we should include an example of that still working here.